modules/era/sql/era_droplist.sql

```diff
@@ -12113,30 +12080,23 @@ INSERT INTO `mob_droplist` VALUES (1477,0,0,1000,1124,@RARE);     -- Wyvern Wing
 INSERT INTO `mob_droplist` VALUES (1478,0,0,1000,2506,310); -- Ladybug Wing (31.0%)
 
 -- ZoneID: 176 - Lagoon Sahagin
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,888,@UNCOMMON);  -- Seashell (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,1428,@UNCOMMON); -- White Mages Testimony (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,624,@RARE);      -- Clump Of Pamtam Kelp (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,887,@RARE);      -- Coral Fragment (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4360,@RARE);     -- Bastore Sardine (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4443,@RARE);     -- Cobalt Jellyfish (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4613,@RARE);     -- Scroll Of Cure V (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4618,@RARE);     -- Scroll Of Curaga Iv (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4638,@RARE);     -- Scroll Of Banish Iii (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4719,@RARE);     -- Scroll Of Regen Iii (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4741,@RARE);     -- Scroll Of Shellra Iv (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4484,@VRARE);    -- Shall Shell (Very Rare, 1%)
-INSERT INTO `mob_droplist` VALUES (1479,2,0,1000,748,0);          -- Gold Beastcoin (Steal)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4514,0);         -- Quus (0.0%)
-INSERT INTO `mob_droplist` VALUES (1479,4,0,1000,4580,0);         -- Coral Butterfly (Despoil)
-INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4703,0);         -- Scroll Of Esuna (0.0%)
+INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,1428,@RARE);  -- White Mages Testimony (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,887,@VRARE);  -- Coral Fragment (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4613,@VRARE); -- Scroll Of Cure V (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4618,@VRARE); -- Scroll Of Curaga Iv (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4638,@VRARE); -- Scroll Of Banish Iii (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4719,@VRARE); -- Scroll Of Regen Iii (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (1479,0,0,1000,4741,@VRARE); -- Scroll Of Shellra Iv (Very Rare, 1%)
```

Comment:
I would put these in a group and make it Uncommon. If you add up the total drops/kills it comes out to approx 18%. Having these all at 1% seems very wrong, as does all the other spells. I know we discussed they should prob be in groups - if you want to make a short list of stuff I can do some drop stuff in retail with TH6 and see how many can drop at oonce. That would solve a lot of these unexplained variances in drop rates.